renderer/reducers/alerts.ts

```diff
@@ -2,9 +2,22 @@
  * Reducer from asynchronous alerts state data.
  */
 
-const asyncAlerts = (state = [], action) => {
+import * as consts from '../consts';
+import { AddAsyncAlertAction, RemoveAsyncAlertAction } from '../types';
+
+type Actions = AddAsyncAlertAction | RemoveAsyncAlertAction;
+
+type AsyncAlertsState = Array<{
+  id?: number;
+  heading?: string;
+  message?: string;
+}>;
+
+const initialState: AsyncAlertsState = [];
+
+export const asyncAlerts = (state: AsyncAlertsState = initialState, action: Actions) => {
```

Comment:
@HaileyJang Here we import the 2 interfaces and combine that as `type Actions`. The `|` is like an `or` and will depending on the action, will only select one of them.

Reply:
Also there's a small bug with the state types here, so currently still fixing that. :)